test/e2e/specs/specs.go

```diff
@@ -1184,10 +1186,20 @@ func CreateImplicitDirInBucket(dirPath, bucketName string) {
 		}
 	}()
 
-	//nolint:gosec
-	if output, err := exec.Command("gsutil", "cp", bucketName, fmt.Sprintf("gs://%v/%v/", bucketName, dirPath)).CombinedOutput(); err != nil {
-		framework.Failf("Failed to create a implicit dir in GCS bucket: %v, output: %s", err, output)
+	// ZB does not currently support gcloud commands, so we use the go GCS client directly.
+	if zbEnabled {
+		dstPrefix := fmt.Sprintf("%s/", dirPath)
+
+		if err := (*storageService).UploadGcsObject(ctx, bucketName, bucketName, dstPrefix, false, zbEnabled); err != nil {
+			klog.Fatalf("CopyBucketToPath failed: %v", err)
```

Comment:
Why do we use klog.Fatalf here, but framework.Failf for non-zonal buckets. 

Should this say UploadGcsObject failed? And not CopyBucketToPath?